tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html

```diff
@@ -323,10 +323,20 @@
       .buttons-container {
         padding: 20px 0;
       }
-      .tensor-data {
-        overflow: auto;
+      #tensor-data {
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        right: 0;
         padding: 20px 0;
```

Comment:
Please add an additional line here:
```css
margin: 0 0 20px 0;
```
This prevents the horizontal scroll bar that some times appears to cause the entire plugin page to jump up and down, which can be distracting during continuation.

Reply:
Done.